egs/babel/s5d/unicode_lexicon_examples.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
It was meant as part of the readme_unicode file, and it seems I forgot to merge those two files. Is it ok if I put these examples in that file? I wanted to put an example of how to add entries to the lexicon from new external text that you get at a later date without having to retrain the acoustic models, but otherwise, most of these examples are in run-1-main-unicode.sh